src/components/UI/Display/Cluster/ClusterIDLabel/index.tsx

```diff
@@ -35,17 +35,26 @@ const Wrapper = styled.span`
   }
 `;
 
-const Label = styled.span`
+const Label = styled.span<{ clusterID: string }>`
   background-color: ${(props) => colorHash.calculateColor(props.clusterID)};
   font-family: ${(props) => props.theme.fontFamilies.console};
   padding: 0.2em 0.4em;
   border-radius: 0.2em;
 `;
 
-const ClusterIDLabel = ({ clusterID, copyEnabled }) => {
+interface IClusterIDLabelProps extends React.ComponentPropsWithoutRef<'span'> {
+  clusterID: string;
+  copyEnabled?: boolean;
+}
+
+const ClusterIDLabel: React.FC<IClusterIDLabelProps> = ({
```

Comment:
Converted this to TypeScript because it was throwing compilation errors after adding props forwarding